app/views/account_shares/_account_share.html.slim

```diff
@@ -3,5 +3,8 @@ tr
   td= account_share.email
   td= account_share.created_at.to_formatted_s(:short)
   td= account_share.accepted_at&.to_formatted_s(:short)
-  td= link_to 'link', accept_account_share_url(token: account_share.token)
+  - if account_share.token.present?
```

Comment:
```suggestion
  - if account_share.public?
```

introduce a method in the model and reverse the results below. with this new method the code becomes closer to the natural language (the business domain)